pandas/tests/extension/test_categorical.py

```diff
@@ -223,6 +223,26 @@ def test_cast_category_to_extension_dtype(self, expected):
 
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "dtype, expected",
+        [
+            (
+                "datetime64[ns]",
+                np.array(["2015-01-01T00:00:00.000000000"], dtype="datetime64[ns]"),
+            ),
+            (
+                "datetime64[ns, MET]",
+                pd.arrays.DatetimeArray(
+                    pd.array([pd.Timestamp("2015-01-01 00:00:00+0100", tz="MET")])
```

Comment:
`pd.DatetimeIndex([pd.Timestamp("2015-01-01 00:00:00+0100", tz="MET")]).array` would be more succint.  would it be any less clear?

Reply:
That's nicer, thanks for the suggestion - I've made the change